app/javascript/components/PublicListings.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <div class='listing-body'>
+      <table>
+        <tr>
+          <th>Created At</th>
+          <th>Name</th>
+          <th>Location</th>
+          <th>Details</th>
+        </tr>
+        <tr v-for="listing in listings" :key="listing.id">
+          <td>{{ listing.created_at }}</td>
+          <td>{{ listing.name }}</td>
+          <td></td>
+          <td></td>
```

Comment:
I have not populated all these fields yet. Do we have a mock-up of what the important fields are?

Reply:
We don't! But this is a great start. I think right now the only thing coming in from our current form is tags. I have some suggestions for additional fields. Best here or in our miro map?